Registry/registry.trad_fields

```diff
@@ -15,7 +15,15 @@ state    real   vmet          ikj   dyn_em    1  -   h1  "VMET"         "earth r
 state    real   speed         ikj   dyn_em    1  -   h1  "SPEED"        "horizontal wind speed"                    "m/s"
 state    real   dir           ikj   dyn_em    1  -   h1  "DIR"          "wind direction, 0=northly, 270=westerly"  "degrees"
 
+state    real   psfc2d        ij    dyn_em    1  -   h1  "SURFACEP"     "surface pressure"                          "Pa"
```

Comment:
@smileMchen @weiwangncar 
If the PSFC field is computed at every time step, and it is used by physics, is it safe to optionally recompute this value in the diagnostics and possibly output the array? If it is safe (specifically: PSFC as computed by physics at the top of a time step is not used as an input variable in the next time step by physics), then we should be able to remove PSFC2d from this Registry (and package), and make sure to add h1 to the PSFC field in the existing Registry file.